### PR TITLE
Adding the ability to create a project via morgue

### PR DESCRIPTION
**Pull request by adamcronin42**
- Added create command to facilitate creating project
- Added to README.md

### CODE REVIEW

**Review by rashkov — APPROVED** [review]
Looks good

File: bin/morgue.js [hunk 1]
```diff
@@ -255,6 +256,62 @@ function coronerError(argv, config) {
...
+  let subcommand = argv._[1];
+
+  var coroner = coronerClientArgv(config, argv);
+  var bpg = coronerBpgSetup(coroner, argv);
+
+  if(!subcommand) {
+    errx("Invalid project command. Try 'morgue project create <your-project-name>'")
+  }
+
+  if(subcommand === 'create') {
```
Comment (konraddysput) [inline bin/morgue.js:271]: small nit:
if you inverse the if statement - you can avoid putting all your code with additional tab.


```suggestion
  if(subcommand !== 'create') {
      errx("Invalid project command. Try 'morgue project create <your-project-name>'")
  }
  let project = argv._[2];
```

File: bin/morgue.js [hunk 2]
```diff
@@ -255,6 +256,62 @@ function coronerError(argv, config) {
...
+      errx("Missing project name");
+    }
+
+    let validationRe = /^[a-zA-Z0-9-]+$/;
+    let validProjName = validationRe.test(project);
+    if(!validProjName) {
+      errx("Illegal name only use a-z, A-Z, 0-9, or \"-\"");
+    }
+
+    if(!config || !config.config) {
```
Comment (konraddysput) [inline bin/morgue.js:283]: small nit - you can put it before the regex. Regex executes some code - those are if statement available since the beginning.

**Review by konraddysput — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -425,6 +425,16 @@ no-op.  The following options affect this behavior:
...
 
+### project
+
+Allows for the creation of projects.
+
+```
+Usage: morgue project create <projectName>
+```
+
+Project names must be in the format of a-z, A-Z, 0-9, or "-".
```
Comment (gittheking) [inline README.md:436]: @adamcronin42 should we reword this to "Project name accepted characters ..."

**Review by gittheking — APPROVED** [review]

File: bin/morgue.js (hunk 1 above)
Comment (adamcronin42) [reply bin/morgue.js:271]: I figured that down the line more subcommands may be added but that is better for the current function.
